package-res/resources/web/pentaho/visual/role/level.js

```diff
@@ -73,24 +78,82 @@ define([
      *    can be (directly) represented by a quantitative visual role.
      */
     return Refinement.extend("pentaho.visual.role.MeasurementLevel", {
-      type: {
+
+      type: /** @lends pentaho.visual.role.MeasurementLevel.Type# */{
+
         id: module.id,
         of: "string",
         facets: ["DiscreteDomain"],
-        domain: ["nominal", "ordinal", "quantitative"],
+        domain: orderedLevels,
 
+        /**
+         * Returns a value that indicates if a given level of measurement is
+         * considered _quantitative_.
+         *
+         * The values of the quantitative levels of measurement are:
+         * 1. `"quantitative"`.
+         *
+         * @param {string|pentaho.type.String} level The measurement level.
+         *
+         * @return {boolean} `true` if it is _quantitative_, `false`, otherwise.
+         */
         isQuantitative: function(level) {
           level = this.to(level);
           return level.value === "quantitative";
         },
 
+        /**
+         * Returns a value that indicates if a given level of measurement is
+         * considered _qualitative_.
+         *
+         * The values of the qualitative levels of measurement are:
+         * 1. `"nominal"`
+         * 2. `"ordinal"`.
+         *
+         * @param {string|pentaho.type.String} level The measurement level.
+         *
+         * @return {boolean} `true` if it is _qualitative_, `false`, otherwise.
```

Comment:
Suggest:

@return {boolean} `true` if it is _qualitative_; `false`, otherwise.